npm/webpack-batteries-included-preprocessor/README.md

```diff
@@ -44,7 +44,7 @@ module.exports = (on) => {
 }
 ```
 
-Other than the `typescript` option, this preprocessor supports the same options as [@cypress/webpack-preprocessor](https://github.com/cypress-io/cypress/tree/develop/npm/webpack-preprocessor#readme), so see its README for more information.
```

Comment:
Just a file change to trigger the release in this package which is triggered by any file update in this package directory. This change is unrelated.